app/screens/transactions/Transactions.tsx

```diff
@@ -187,17 +193,28 @@ const Transactions = ({ history }: RouteComponentProps) => {
     (state: RootState) =>
       state.wallet.accounts[state.wallet.currentAccountIndex].address
   );
+
+  const getNonce = useSelector(
```

Comment:
Btw, it would be better to rename it into `nonce`, since it is not a function. But it is a minor one.